src/main.ts

```diff
@@ -552,8 +552,9 @@ async function main() {
       'Run providers interactively, one at a time',
       defaultConfig?.evaluateOptions?.interactiveProviders,
     )
-    .option('-n, --first-n <number>', 'Only run the first N tests')
-    .option('--pattern <pattern>', 'Only run tests whose description matches the regular expression pattern')
+    .option('-n, --filter-first-n <number>', 'Only run the first N tests')
```

Comment:
This will be a breaking change so will need to do a version bump for the next release.

Reply:
My selfish preference is to keep `-n` because it feels familiar, like `head -n` :).  Open to other short forms but I don't think it's necessary.  If we find ourselves getting tired of typing everything out let's add it separately.

I do version bumps manually - since we're pre-1.0 I've included breaking changes in minor versions, but generally trying to avoid them.  I think this is acceptable though.  I'll merge with `feat!` breaking change notation and make a note in the release notes.